recipes/conversation_view/emoji_row/emoji_row.stories.js

```diff
@@ -0,0 +1,98 @@
+import { action } from '@storybook/addon-actions';
+import { createTemplateFromVueFile } from '@/common/storybook_utils';
+import DtRecipeEmojiRow from './emoji_row.vue';
+import DtRecipeEmojiRowDefaultTemplate from './emoji_row_default.story.vue';
+
+// Default Prop Values
+export const argsData = {
+  onEmojiClicked: action('emojiClicked'),
+  onEmojiHovered: action('emojiHovered'),
```

Comment:
These actions aren't being output in the storybook.. I'm actually not sure why.. looks like you did it correctly. Are they maybe not actually emitting from the component?

Reply:
Its a naming issue...

in the stories.js the events are registered like this:
onEmojiClicked: action('`emojiClicked`'),
onEmojiHovered: action('`emojiHovered`'),

But on emoji_row_default.story.vue are like this:
@`emoji-clicked`="onEmojiClicked"
@`emoji-hovered`="onEmojiHovered"

If you update it like
@`emojiClicked`="onEmojiClicked"
@`emojiHovered`="onEmojiHovered"

It will work but linter will piss off because events must be hyphenated and he is right.

My suggestions here is update the event names to be hyphernated.

Reply:
looks like it's working now